Marlin/boards.h

```diff
@@ -52,9 +52,11 @@
 #define BOARD_HEPHESTOS         42   // bq Prusa i3 Hephestos
 #define BOARD_BAM_DICE          401  // 2PrintBeta BAM&DICE with STK drivers
 #define BOARD_BAM_DICE_DUE      402  // 2PrintBeta BAM&DICE Due with STK drivers
+#define BOARD_FELIX2			101  // Felix 2.0+ Electronics
```

Comment:
I'm not sure how we come up with these numbers. But perhaps make it "related" to other RAMPS 1.3 derived printers' number codes. Vague, I know. Something beginning with "3" or "4" seems to be the current trend.

Also consider adding printer name (and firmware info) to the `language.h` file. You can leave the firmware_url unset if you just want MarlinFirmware/Marlin to be the firmware source.

Also, you should add your FELIX-specific configuration files to the `example_configurations` folder. Make a sub-folder there named "FELIX" and put your Configuration files there. Make sure they have all the latest additions from the current configuration files.